azerothcore/azerothcore-wotlk
# fix(DB/npc_text): The Stone Watcher

## Changes Proposed:
-  Swap id's inside  npc_text.

## Issues Addressed:

- Closes [Various issue regarding to The Stone Watcher quest #9508 ](https://github.com/azerothcore/azerothcore-wotlk/issues/9508)
- Closes https://github.com/chromiecraft/chromiecraft/issues/2475

## SOURCE:

[The Stone Watcher](https://wowpedia.fandom.com/wiki/The_Stone_Watcher?oldid=1869106)

## Tests Performed:

- Added quest [Seeing What Happens](https://classic.wowhead.com/quest=2946/seeing-what-happens) (.q add 2946).
- Submitted quest and NPC spawns and gives u another quest [The Stone Watcher](https://classic.wowhead.com/quest=2954/the-stone-watcher)
- Accepted quest and clicked on gossips and made sure they in right order.


## How to Test the Changes:


1. Clean wow cache.
2. Add quest [Seeing What Happens](https://classic.wowhead.com/quest=2946/seeing-what-happens) (.q add 2946).
3. Submit quest and NPC spawns and gives u another quest [The Stone Watcher](https://classic.wowhead.com/quest=2954/the-stone-watcher)
4. Accept quest. Click and check that info in gossips matches info from Source.

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
